perception/autoware_tensorrt_bevdet/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+cmake_minimum_required(VERSION 3.17)
+project(autoware_tensorrt_bevdet)
+
+add_compile_options(-W)
+add_compile_options(-std=c++17)
+set(CMAKE_CXX_FLAGS_RELEASE "-Wno-deprecated-declarations -O2")
```

Comment:
The optimization level `-O2` in `CMAKE_CXX_FLAGS_RELEASE` is a good default, but consider providing options for different optimization levels (e.g., `-O0`, `-O3`) using CMake's build type settings. This allows for more flexibility during development and testing.